jsdrupal/drupal-admin-ui
# Nightwatch Test for admin/content

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/174

## Screenshot / UI changes

N/A

## Testing instructions

Run `yarn nightwatch` – test should wait for the select for content type to appear, click it, and ensure that "article", "page", and "recipe" are visible in the corresponding popover.
